src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/ServiceController.php

```diff
@@ -83,8 +84,17 @@ public function overviewAction()
         if (empty($services)) {
             return $this->redirectToRoute('service_add');
         }
+        // If only one service is available and you are admin, select that one
+        if ($this->isGranted('ROLE_ADMINISTRATOR') && count($services) === 1) {
+            $service = reset($services);
+            $this->authorizationService->changeActiveService($service->getId());
+            return $this->redirect(
+                $this->generateUrl('service_admin_overview', ['serviceId' => $service->getId()])
+            );
+        }
```

Comment:
Scenario when only one service is present in the SP dashboard
And you log in as an `admin` user
Then you should be auto-forwarded to the service overview page of that one service

Previously you would land on the 'select a service' admin welcome page. However, the service switcher would not be displayed as only one service is present.